src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,92 @@
+// ----- BASE ELEMENTS ---------------
+
+.navbar {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
+  background-color: #000000;
+}
+
+.img-left {
+  float: left;
+  width: 159px;
+  height: 56px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.right-menu {
+  float: right;
+}
+
+.navbar a img {
+  width: 119px;
+  height: 24px;
+}
+
+.navbar a {
+  float: left;
+  font-size: 16px;
+  color: #ffffff;
+  text-align: center;
+  padding: 1rem;
+  text-decoration: none;
+  line-height: 1;
+  height: 56px;
+}
+
+.navbar p {
+  color: #ffffff;
+  line-height: 1;
+}
+
+.dropdown {
+  float: right;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
+  font-size: 16px;
+  border: none;
+  outline: none;
+  color: #ffffff;
+  padding: 14px 16px;
+  background-color: inherit;
+  font-family: inherit;
+  margin: 0;
+  height: 56px;
+}
+
+.navbar a:hover, .dropdown:hover .dropbtn {
```

Comment:
Please change to `.eyebrow ...`.  This class name may change, so we'll update if needed.

Reply:
Changes have been done, Please review